src/plugins/controls/timer/plugin.js

```diff
@@ -175,11 +187,14 @@ export default pluginFactory({
                             const timers = self.timerbox.getTimers();
 
                             const updatedTimers = Object.keys(timers).reduce((acc, timerName) => {
+                                const stats = statsHelper.getInstantStats(timers[timerName].scope, testRunner);
```

Comment:
if scope is not supported by `getInstantStats`, it will return `test` scope. `item` scope is not supported, so, if you will try to get `stats` for `item` scope, it will return `test` scope stats. please consider it and generate proper message for `item` scope timer